pkgs/by-name/ch/choose-gui/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, system, fetchFromGitHub, xcbuild, coreutils, darwin }:
```

Comment:
I suggest you to use [nixfmt-rfc-style](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=nixfmt) formatter. Because is the default formatter for nixpkgs currently.